### PR TITLE
Footer 

### PR DESCRIPTION
**Pull request by mattgle**
## Summary
Footer

## Trello Card
https://trello.com/c/7VM7RLQa/14-footer

## Screenshots
captura de pantalla 2018-10-26 a la s 14 36 30

### CODE REVIEW

File: maquetado/index.html [hunk 1]
```diff
@@ -78,5 +78,10 @@ <h4 class="tech-release">Release: 2005</h4>
       </div>
     </div>
   </div>
+  <div class="footer">
+      <h4 class="footer-title">WE ARE GAME CHANGERS</h4>
```
Comment (henryzarza) [inline maquetado/index.html]: Remember the spacing are 2

File: maquetado/index.html [hunk 2]
```diff
@@ -78,5 +78,10 @@ <h4 class="tech-release">Release: 2005</h4>
...
+      <img src="assets/wolox_logo_white.svg" alt="wolox logo white" class="footer-wolox-icon">
```
Comment (henryzarza) [inline maquetado/index.html]: The same here

File: maquetado/index.html [hunk 3]
```diff
@@ -78,5 +78,10 @@ <h4 class="tech-release">Release: 2005</h4>
...
+      <h6 class="footer-text">Copyright 2018. Wolox. All rights reserved</h6>
```
Comment (henryzarza) [inline maquetado/index.html]: And here

File: maquetado/scss/application.scss [hunk 4]
```diff
@@ -1,8 +1,9 @@
 @import 'equalizer';
-@import 'variables';
-@import 'colors';
+@import 'variables/variables';
+@import 'variables/colors';
 @import url('https://fonts.googleapis.com/css?family=Roboto');
```
Comment (henryzarza) [inline maquetado/scss/application.scss]: Move this above

File: maquetado/scss/footer.scss [hunk 5]
```diff
@@ -0,0 +1,20 @@
...
+  color: $white;
+  font-size: $text-big;
+  margin-bottom: 14px;
+}
+
+.footer-text {
+  align-self: flex-end;
+  color: $white;
+  font-size: $text-small;
+}
```
Comment (henryzarza) [inline maquetado/scss/footer.scss:20]: I think that is necessary `margin-top` property

File: maquetado/scss/tech.scss [hunk 6]
```diff
@@ -50,7 +51,7 @@
     left: -10px;
     position: absolute;
     top: -10px;
-    transform: all 0.5s ease;
+    transition: all 0.5s ease;
```
Comment (henryzarza) [inline maquetado/scss/tech.scss]: Change this for `transition: transform 0.5s ease;`

**Review by henryzarza — CHANGES_REQUESTED** [review]

File: maquetado/scss/footer.scss [hunk 7]
```diff
@@ -0,0 +1,21 @@
...
+  color: $white;
+  font-size: $text-big;
+  margin-bottom: 14px;
+}
+
+.footer-text {
+  align-self: flex-end;
+  color: $white;
+  font-size: $text-small;
+  margin-top: 5px;
```
Comment (henryzarza) [inline maquetado/scss/footer.scss]: Change for `14px`

File: maquetado/scss/application.scss [hunk 8]
```diff
@@ -1,8 +1,9 @@
 @import 'equalizer';
-@import 'variables';
-@import 'colors';
-@import url('https://fonts.googleapis.com/css?family=Roboto');
+@import 'variables/variables';
+@import 'variables/colors';
 @import 'styles';
+@import url('https://fonts.googleapis.com/css?family=Roboto');
```
Comment (henryzarza) [inline maquetado/scss/application.scss]: Move this above

**Review by henryzarza — CHANGES_REQUESTED** [review]

**Review by henryzarza — APPROVED** [review]

File: maquetado/scss/footer.scss [hunk 9]
```diff
@@ -0,0 +1,21 @@
...
+  color: $white;
+  font-size: $text-big;
+  margin-bottom: 14px;
+}
+
+.footer-text {
+  align-self: flex-end;
+  color: $white;
+  font-size: $text-small;
+  margin-top: 14px;
```
Comment (SKOLZ) [inline maquetado/scss/footer.scss]: It's preferred to add this as a margin bottom in the footer-wolox-icon, to be consistent with the rest and also to make it easier to read. When you check the html in the inspector you can easily see that the image is separated from the text by 14px at the bottom.

**Review by SKOLZ — CHANGES_REQUESTED** [review]

**Review by SKOLZ — APPROVED** [review]

**Pull request merged**